src/Microsoft.AspNet.SignalR.SqlServer/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+[assembly: AssemblyTitle("Microsoft.AspNet.SignalR.SqlServer")]
+[assembly: AssemblyDescription("Assembly containing components for scaling-out SignalR applications using SQL Server.")]
```

Comment:
No need for the  `AssemblyTitle` or `AssemblyDescription` attributes anymore. The only thing you need is `InternalsVisableTo`.